src/player.js

```diff
@@ -715,6 +715,29 @@ export default class Player extends FakeEventTarget {
     }
   }
 
+  /**
+   * Get the live relative time in seconds.
+   * @returns {?Number} - The live relative time.
+   * @public
+   */
+  get liveTime(): number {
+    if (this.isLive()) {
```

Comment:
This condition already exists in the engine

Reply:
fixed